src/theme/toolbar.m.css

```diff
@@ -1,5 +1,14 @@
-.actions { }
-.collapsed { }
+.actions {
+	display: flex;
+	flex-direction: row;
+}
+
+.collapsed {
+	& .actions {
```

Comment:
sorry to be a pain, but we're avoiding the use of nested selectors via the `&` operator. Could you change this to `.collapsed .actions` please.

Reply:
Changed in bf161c0